doc/bn.tex

```diff
@@ -2604,13 +2604,13 @@ \section{Function Macros}
 \begin{alltt}
 bool mp_iseven(const mp_int *a)
 \end{alltt}
-Checks if $a = 0 mod 2$
+Checks if $a = 0 \;\mathrm{mod}\; 2$
```

Comment:
Why not `\mod`? Ah, I see: `\def\mod{{\mathit\ mod\ }}` that's not the most elegant solution. It is used as the symbol for "remainder" so I would propose `\def\mod#1{\allowbreak\mkern18mu{\rm mod}\,\,#1}` instead, maybe with a bit less kerning.

```
\documentclass{article}
% The definition of "\pmod" from plain TeX without the parentheses.
\def\mod#1{\allowbreak\mkern18mu{\rm mod}\,\,#1}
\def\tightmod#1{\allowbreak\mkern12mu{\rm mod}\,\,#1}
\begin{document}
With \texttt{\textbackslash mod}.
$$ a = 0 \mod{2}$$
With \texttt{\textbackslash tightmod} which has a third less
space before ``mod''.
$$ a = 0 \tightmod{2}$$
No extra packages used.
\end{document}
```
There is also a curious redefinition of `\pmod` which is nowhere to be seen in use. It is, well should be, a `\mod` with parentheses to denote an  equivalence class. I haven't taken a deeper look into ``bn.tex` for some time but it (the original with parentheses) could be useful in some places.

Reply:
I'm going to merge this and after your latest is merged I'll have a look over the file. I already started changing stuff